src/lang/std/artifactGraph.ts

```diff
@@ -98,13 +98,19 @@ interface WallArtifact extends BaseArtifact {
   edgeCutEdgeIds: Array<ArtifactId>
   sweepId: ArtifactId
   pathIds: Array<ArtifactId>
+  // codeRef is for the sketchOnFace plane, not for the wall itself
+  // traverse to the extrude and or segment to get the wall's codeRef
+  codeRef?: CodeRef
```

Comment:
We should rename this to `planeCodeRef` or something. It's too confusing if it's named `codeRef`, and for everything else, it's the codeRef of the artifact itself.